Makefile

```
@@ -497,32 +497,34 @@ kind-clustermesh-images: kind-clustermesh-ready kind-build-clustermesh-apiserver
 	$(QUIET)kind load docker-image $(LOCAL_KVSTOREMESH_IMAGE) --name clustermesh1
 	$(QUIET)kind load docker-image $(LOCAL_KVSTOREMESH_IMAGE) --name clustermesh2
 
+ENABLE_KVSTOREMESH=false
```

Should you leave the possibility to change it from the environment?

```suggestion
ENABLE_KVSTOREMESH ?= false
```